pkg/reconciler/v1alpha1/pipelinerun/pipelinerun.go

```diff
@@ -164,7 +168,11 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 	pr := original.DeepCopy()
 	pr.Status.InitializeConditions()
 
-	if isDone(&pr.Status) {
+	if pr.Status.IsDone() {
+		statusMapKey := fmt.Sprintf("%s/%s", pipelineRunControllerName, key)
```

Comment:
We may want to use `getPipelineRunKey` (same for `TaskRun`) to make sure we use the same key always :angel: